src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -434,7 +433,7 @@ static bool _appendClipShape(SvgLoaderData& loaderData, SvgNode* node, Shape* sh
         }
     }
 
-    _applyProperty(loaderData, node, shape, vBox, svgPath, true);
+    _applyComposition(loaderData, shape, node, vBox, svgPath);
```

Comment:
This change does not affect the transformation of clips.
It will be reviewed again along with #3082.